samples/drivers/display/src/main.c

```diff
@@ -211,13 +180,12 @@ void main(void)
 	size_t buf_size = 0;
 	fill_buffer fill_buffer_fnc = NULL;
 
-	LOG_INF("Display sample for %s", DISPLAY_DEV_NAME);
-
-	display_dev = device_get_binding(DISPLAY_DEV_NAME);
+	LOG_INF("Display sample for %s", DT_LABEL(DT_CHOSEN(zephyr_display)));
 
-	if (display_dev == NULL) {
+	display_dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_display));
+	if (!device_is_ready(display_dev)) {
 		LOG_ERR("Device %s not found. Aborting sample.",
-			DISPLAY_DEV_NAME);
+			DT_LABEL(DT_CHOSEN(zephyr_display)));
```

Comment:
`display_dev->name`. Here and elsewhere in this patch, getting the name from the device struct seems a little cleaner.